draft-ietf-tls-tls13.md

```diff
@@ -4319,6 +4322,25 @@ will receive various TLS 1.x versions in this field, however its value
 MUST always be ignored.
 
 
+## Zero-RTT backwards compatibility
+
+A TLS 1.3 ClientHello with 0-RTT data is not compatible with older servers.
+This can cause compatibility issues if a client attempts to use 0-RTT with a
```

Comment:
maybe just "issues" since the previous sentence told us they were compat issues.